data/mods/Aftershock/mobs/aliens.json

```diff
@@ -151,5 +151,39 @@
     "anger_triggers": [ "STALK", "FRIEND_ATTACKED", "FRIEND_DIED", "PLAYER_WEAK", "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PATH_AVOID_DANGER_1", "WARM", "KEENNOSE" ]
+  },
+  {
+    "type": "MONSTER",
+    "id": "afs_titanis",
+    "name": { "str": "titanis" },
```

Comment:
My only request here is if you could think to 0give it a more "avian" name. Not sure what it could be, you are probably more versed in Dinosaur  names but it could be something -raptor, or maybe the name of some  artic dinosaur if those existed

Reply:
Titanis is in the Terror Bird family, which is pretty great and unmistakably birdlike. How about that? 
As far as arctic dinosaurs, yes there were some. I've been thinking about doing Troodons or Stenonychosaurus for a long time. From a game design perspective they were pretty similar to Titanis, on the smaller side, predatory, but not suicidal, fed on smaller prey. Thought to have been way smarter than most dinos, though I'm not sure our AI is good enough to model that.
![image](https://user-images.githubusercontent.com/26608431/125177642-7136b500-e1ab-11eb-8ca1-7af9f0405fbb.png)

Reply:
Yep seems great thank you!